netmaster/objApi/apiController.go

```diff
@@ -1571,6 +1571,38 @@ func (ac *APIController) RuleCreate(rule *contivModel.Rule) error {
 		return core.Errorf("Policy not found")
 	}
 
+	if rule.Direction == "in" && rule.ToIpAddress != "" {
+		// rules from k8s network policy
+		// verify 'toIpAddress' is part of epg and subnet
+		if len(policy.LinkSets.EndpointGroups) <= 0 {
+			errMsg := fmt.Errorf("no linked endpoint groups exists to use --to-ip-address")
```

Comment:
Sorry, I didn't notice this in the first review.  These can just be a regular string literal.  Nothing is being interpolated.

```go
errMsg := "no linked endpoint groups exists to use --to-ip-address"
```

Same thing in the other three spots

Reply:
nope, error type is returned not string.

Reply:
Ah yes,  you can use

```go
return errors.New(errMsg)
```

when you don't need a format string.  One of the code quality tools (I forget which one) complains if you use a format string function with no variables being interpolated